static/app/components/stream/group.tsx

```diff
@@ -415,6 +419,17 @@ function BaseGroupRow({
     </DeprecatedDropdownMenu>
   );
 
+  const lastTriggered = !defined(lastTriggeredDate) ? (
+    <Placeholder height="18px" />
```

Comment:
does this need a loading placeholder or is the stat available as soon as it renders?

Reply:
It should be available when it renders, it's returned from the same endpoint as the groups themselves